app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTile.kt

```diff
@@ -47,8 +62,8 @@ fun SettingsWearTemplateTile(
                 docsLink = WEAR_DOCS_LINK
             )
         }
-    ) {
-        Column(Modifier.padding(all = 16.dp)) {
+    ) { padding ->
+        Column(Modifier.padding(padding).padding(all = 16.dp)) {
```

Comment:
If not using the padding of the `Scaffold`, then the entire block becomes red in AS. However, the `Scaffold` padding does not give the right margin all around...

Reply:
I believe this was a change in lint, there are lots of `Scaffold`s without using the padding in the codebase and they all show up in red now (but you can still compile the app without problems).

A cleaner solution while stil ensuring that the required `Scaffold` padding is used would be:
- calculating the new totals using `calculateStartPadding` etc from the parent and then `coerceAtLeast(16)` for all 4 sides, for example: `padding.calculateStartPadding(LocalLayoutDirection.current).value.coerceAtLeast(16f).dp`
- switching to a `LazyColumn` and using the parent value for the modifier, and setting your existing padding of 16 dp as the `contentPadding`

Reply:
'Stacking' modifiers like this works so I'd say the option you've already implemented is a good fit for this screen 👍

Reply:
Alright! Thanks for you feedback 🙂